clusterloader2/cmd/clusterloader.go

```diff
@@ -55,13 +55,20 @@ func initClusterFlags() {
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.Provider, "provider", "PROVIDER", "", "Cluster provider")
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.MasterName, "mastername", "MASTER_NAME", "", "Name of the masternode")
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.MasterIP, "masterip", "MASTER_IP", "", "Hostname/IP of the masternode")
+	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.KubemarkRootKubeConfigPath, "kubemark-root-kubeconfig", "DEFAULT_KUBECONFIG", "",
```

Comment:
s/DEFAULT_KUBECONFIG/KUBEMARK_ROOT_KUBECONFIG/

default is misleading and inconsistent with all other names in this line

Reply:
I totally agree, but the DEFAULT_KUBECONFIG already exists and does exactly what we need (see e.g. https://github.com/kubernetes/kubernetes/blob/c9e14c85f5b826493a2376ab4743499ac834b8ee/test/kubemark/run-e2e-tests.sh#L42)

In that case I decided to use it instead of introducing another env variable. 

But I'm happy to change it to KUBEMARK_ROOT_KUBECONFIG (+ setting it in the cluster set-up scripts).
LMK WDYT

Reply:
OK - if it already exists, I can live with it.

Please just add a TODO to fix that later so that we don't forget why this is like this.

Reply:
Done.